### PR TITLE
Provisional solution for the search-radius for SpreadDEM

### PR DESCRIPTION
**Pull request by GuillermoCasas**

### CODE REVIEW

**Comment by GuillermoCasas** [comment]
All the DEM tests pass (the run_all_benchmarks.py). One of the results files has been altered to the new solution, which is considered to be more accurate, since a larger search radius is used now (example 16). This is a provisional arrangement that we need to have a working solution for our presentation of SpreadDEM.

**Review by maceligueta — CHANGES_REQUESTED** [review]
Too many files! Someone uploaded the results?

**Comment by maceligueta** [comment]
I removed a lot of files

File: applications/DEM_application/custom_strategies/strategies/explicit_solver_strategy.cpp [hunk 1]
```diff
@@ -381,30 +381,6 @@ namespace Kratos {
...
-                        for (unsigned int j = 0; j < mListOfSphericParticles[i]->mNeighbourRigidFaces.size(); j++) {
-                            DEMWall* p_wall = mListOfSphericParticles[i]->mNeighbourRigidFaces[j];
-                            #pragma omp critical
-                            {
-                                p_wall->mNeighbourSphericParticles.push_back(mListOfSphericParticles[i]);
-                            }
-                        }
-                    }
-                }//#pragma omp parallel
-
```
Comment (maceligueta) [inline applications/DEM_application/custom_strategies/strategies/explicit_solver_strategy.cpp]: Why do you think this is not necessary? If the neighbour walls change, the walls should also be aware that the neighbour particles change.

Comment (GuillermoCasas) [reply applications/DEM_application/custom_strategies/strategies/explicit_solver_strategy.cpp]: Yes, it is a mistake that I made thinking about the previous design. Now the neighbours do change potentially at every step.

**Pull request closed without merging**